django_prices/widgets.py

```diff
@@ -26,14 +24,15 @@ class FixedCurrencyMoneyInput(forms.MultiWidget):
 
     def __init__(self, currency, attrs=None):
         self.currency = currency
-        widgets = [forms.TextInput(attrs={"type": "number", "step": "any"})]
+        widgets = [
+            forms.TextInput(attrs={"type": "number", "step": "any"}),
+            forms.HiddenInput(),
```

Comment:
Shouldn't we somehow indicate which currency we're using?

Reply:
Field receives currency value from form and rendering can be altered using template